app/emoji-store/service.js

```diff
@@ -24,7 +42,10 @@ export default Ember.Service.extend({
   seedEmoji() {
     for (let name of Object.keys(emojilib.lib)) {
       let {char, category, keywords} = emojilib.lib[name];
-      let emoji = {name: humanize(name), char};
+      let emoji = {
+        name: (category === 'flags')?countryName(name.toUpperCase()):humanize(name),
```

Comment:
This might prove tricky because:

1) The `humanize` function doesn't have access to the `category` value
2) Some emoji names clash with valid country codes (like :cd: for example)